### PR TITLE
First draft manifest file post

### PR DESCRIPTION
**Pull request by finestructure**
First draft, opening PR for discussion/editing

### CODE REVIEW

File: Content/posts/swift-package-index-metadata-file.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    scheme: ComposableArchitecture_watchOS
+```
+
+Our builders are looking for this file and use the scheme indicated for the given platform. If a platform is not listed we use the default heuristics to determine the scheme as outlined in our [build FAQ][4].
+
+### Images for Linux
+
+Another use case for the `.spi.yml` manifest file is to configure custom base images for our Linux builds.
+
+For our Linux build we run docker commands for various Swift version base images:
```
Comment (finestructure) [inline Content/posts/swift-package-index-metadata-file.md:39]: When building for Linux, we run ...

File: Content/posts/swift-package-index-metadata-file.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+```bash
+/usr/local/bin/docker run --rm -v "$PWD":/host -w /host swiftlang/swift:5.2.4 swift build --enable-test-discovery
+```
+
+This works great except when a package requires an OS level dependency to be available on the system. A common example is OpenSSL.
```
Comment (finestructure) [inline Content/posts/swift-package-index-metadata-file.md:45]: unless a package requires

File: Content/posts/swift-package-index-metadata-file.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+While we may eventually provide our own set of base images for each supported Swift version, we currently ask package authors whose builds fail with a plain Swift base image to create their own base images for the Swift versions they support.
```
Comment (finestructure) [inline Content/posts/swift-package-index-metadata-file.md:47]: base image much? 😆

File: Content/posts/swift-package-index-metadata-file.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+You will also need to inform us when you do so, because for security reasons we don’t allow arbitrary images and therefore need to add your images to our allow list.
```
Comment (finestructure) [inline Content/posts/swift-package-index-metadata-file.md:49]: we don't support arbitrary images

File: Content/posts/swift-package-index-metadata-file.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+---
+date: 2020-10-30 12:00
+title: The Swift Package Index Metadata File – First Steps
+description: …
+---
+
+The vast majority of the information you see in [the package index](https://swiftpackageindex.com) is generated from a single URL, the location of the package’s git repository.
+
+From that URL, we’re able to gather a huge amount of metadata. We gather data from the repository itself, the package manifest, the GitHub API (yes, it’s GitHub only for now, but [we do plan to support other hosts](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/485)), and from running builds of the package to check compatibility with operating systems and Swift versions.
```
Comment (finestructure) [inline Content/posts/swift-package-index-metadata-file.md:9]: I feel like `huge amount` sounds a bit big, perhaps `a lot of`? Just a nit though.

Comment (daveverwer) [reply Content/posts/swift-package-index-metadata-file.md]: Yep, the whole sentence was a bit unnecessary actually. Changed to:

> From that URL we gather data from the repository itself, ...

**Review by daveverwer — APPROVED** [review]

**Pull request merged**